src/plugins/data/public/search/search_interceptor/search_interceptor.ts

```diff
@@ -462,6 +463,10 @@ export class SearchInterceptor {
     }
   };
 
+  public showWarning({ title, text }: ResponseWarning) {
```

Comment:
this seems too generic to belong to search interceptor

Reply:
On the other hand, interceptor does basically the same thing with `showError`